cl/alerts/tasks.py

```diff
@@ -168,25 +200,58 @@ def send_docket_alert(
         delete_redis_semaphore("ALERTS", make_alert_key(d_pk))
         return
 
-    # Notify every user that's subscribed to this alert.
-    messages = make_alert_messages(
-        d, new_des, email_addresses, recap_email_recipients
+    docket_alert_messages = make_alert_messages(
+        d,
+        new_des,
+        docket_alert_recipients,
+        first_email=False,
+        auto_subscribe=False,
+    )
+    re_first_time_auto_subscribe_messages = make_alert_messages(
+        d,
+        new_des,
+        recap_email_auto_subscribe_recipients,
+        first_email=True,
+        auto_subscribe=True,
+    )
+    re_first_time_no_auto_subscribe_messages = make_alert_messages(
+        d,
+        new_des,
+        recap_email_no_auto_recipients,
+        first_email=True,
+        auto_subscribe=False,
+    )
+
+    messages = (
+        docket_alert_messages
+        + re_first_time_auto_subscribe_messages
+        + re_first_time_no_auto_subscribe_messages
     )
```

Comment:
After the refactor I mention above, I think this will all drop away. Aaaaaahhhh.

Reply:
Yeah, this was removed!